R/daedalus.R

```diff
@@ -256,8 +256,11 @@ daedalus <- function(country,
   ) # coerce to logical; automatically FALSE as default value is 0.0
 
   if (!is_response_active) {
-    rlang::env_poke(
-      parameters[["mutables"]], "closure_time_start", response_time
+    # set switch parameter and log closure start time if not 0.0/FALSE
```

Comment:
```suggestion
    # set switch parameter and log closure start time if not 0.0 or FALSE
```

I thought this was about NaN for a minute